### PR TITLE
Clarify docs in misc/__init__.py

### PR DESCRIPTION
**Pull request by BlGene**
Clarified the fact that some functions will not be avaliable without PIL, this was not clear before.

BR, Max

### CODE REVIEW

File: scipy/misc/__init__.py [hunk 1]
```diff
@@ -8,33 +8,33 @@
 Various utilities that don't have another home.
 
 Note that the Python Imaging Library (PIL) is not a dependency
-of SciPy and therefore the `pilutil` module is not available on
-systems that don't have PIL installed.
+of SciPy, but the `pilutil` module and the image manipulation
```
Comment (rgommers) [inline scipy/misc/__init__.py]: While you're at it, can you remove ``the `pilutil` module``? It's not public so shouldn't be mentioned here.

File: scipy/misc/__init__.py [hunk 2]
```diff
@@ -8,33 +8,33 @@
 Various utilities that don't have another home.
 
 Note that the Python Imaging Library (PIL) is not a dependency
```
Comment (rgommers) [inline scipy/misc/__init__.py]: Can you change ``Python Imaging Library (PIL)`` to ``Pillow (https://python-pillow.org/)``? PIL shouldn't be used anymore, it's abandonware.

Comment (rgommers) [inline scipy/misc/__init__.py]: And then ``[requires Pillow]`` a bunch of times below.

**Comment by rgommers** [comment]
Thanks @BlGene, these changes make sense to me.

**Comment by BlGene** [comment]
@rgommers Thanks for the quick reply, it should be ready now.

BR, Max

**Comment by rgommers** [comment]
Merged. Thanks Max.

**Pull request merged**